semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser_cli_pattern/error.txt

```diff
@@ -0,0 +1,7 @@
+error: invalid pattern
+  --> CLI Input:1
+1 | #include $X
+  | ^^^^^^^^^^
+
+Pattern could not be parsed as C
```

Comment:
Maybe be more clear in that it can't be parsed as a C semgrep pattern so the user doesnt get confused why a pattern is being parsed as code